modules/installation-special-config-raid.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing_customizing.adoc
+
+[id="installation_special-config-raid_{context}"]
+= Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume.
+
+.Procedure
+
+. Create a machine config that configures a data volume by using software RAID in the `<installation_directory>/openshift` directory. In this example, it is called `raid1-alt-storage.yaml`:
++
+.Sample RAID 1 on secondary disks configuration file
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: raid1-alt-storage
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    storage:
+      disks:
+      - device: /dev/sdc
+        partitions:
+        - label: data-1
+        wipeTable: true
+      - device: /dev/sdd
+        partitions:
+        - label: data-2
+        wipeTable: true
+      filesystems:
+      - device: /dev/md/md-data
+        format: xfs
+        path: /var/lib/containers
+        wipeFilesystem: true
+      raid:
+      - devices:
+        - /dev/disk/by-partlabel/data-1
+        - /dev/disk/by-partlabel/data-2
+        level: raid1
+        name: md-data
+    systemd:
+      units:
+      - contents: |-
+          [Unit]
+          Before=local-fs.target
+          Requires=systemd-fsck@dev-md-md\x2ddata.service
+          After=systemd-fsck@dev-md-md\x2ddata.service
+
+          [Mount]
+          Where=/var/lib/containers
+          What=/dev/md/md-data
+          Type=xfs
+
+          [Install]
+          RequiredBy=local-fs.target
+        enabled: true
+        name: var-lib-containers.mount <1>
+----
++
+<1> If you choose a different mount point, you must update the unit name to correspond to your mount point. Otherwise the unit will not activate. You can generate a matching unit name with `echo $(systemd-escape -p $mountpoint).mount` where `$mountpoint` is your chosen mount point.
++
+. Apply the config to your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc create -f raid1-alt-storage.yaml
```

Comment:
This doesn't match the mention of `<installation_directory>/openshift` at the top of the procedure, which implies that we're creating the MachineConfig before installing a new cluster instead of applying it to an existing cluster.  Also, as it turns out, we can _only_ configure RAID before installing a new cluster; see [2066913](https://bugzilla.redhat.com/show_bug.cgi?id=2066913).

Reply:
thanks for bringing this to my attention, I'll have this bug assigned to me and work on updating things in a new PR.